lib/ansible/modules/apt.py

```diff
@@ -218,6 +228,12 @@
     default_release: squeeze-backports
     update_cache: yes
 
+- name: Install the version '1.18.0' of package "nginx" and allow potential downgrades
+  apt:
+    name: nginx=1.18.0
+    state: present
+    allow_downgrade: yes
```

Comment:
```suggestion
    allow_downgrade: true
```

Reply:
All the examples in the apt module documentation use yes/no booleans rather than true/false, why are you suggesting that this be changed?

Reply:
We prefer `yes/no` in documentation fwiw.

Reply:
The reason I mentioned this to be discussed is: https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics
> Use lowercase ‘true’ or ‘false’ for boolean values in dictionaries if you want to be compatible with default yamllint options.

But if maintainers prefer `yes/no` feel free to ignore my suggestion.